### PR TITLE
feat:  Added Card Component

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
different types of cards are added
- Basic Card
- Card with Buttons
- Horizontal card
- Ecommerce Card
- Text Overlay card
- Card with Badge

### CODE REVIEW

**Review by nerdyspook — APPROVED** [review]
Code looks good to merge.

File: Components/Card/card.html [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+
+            </div>
+
+            <!-- ---------------------------------------------------------------- -->
+            <p class="padding-l">To use the above basic card components, just copy and paste the below code snippets:</p>
+            <iframe src="https://www.thiscodeworks.com/embed/6203eb9ec6efb50015062d6f" style="width: 100%; height: 242px;" frameborder="0"></iframe>
+
+
+
+
```
Comment (shivanigangadharan) [inline Components/Card/card.html]: Can remove extra spaces here

File: Components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    .txt-overlay {
+        position: relative;
+    }
+    
+    .txt-container {
+        border-radius: 10px;
+        position: absolute;
+        height: 100%;
+        width: 100%;
+        background-color: rgb(104, 104, 103);
```
Comment (shivanigangadharan) [inline Components/Card/card.css]: Can use a color var here too

**Review by shivanigangadharan — APPROVED** [review]
Cards look nice, code looks good to merge.

File: Components/Card/card.html (hunk 1 above)
Comment (MaruthiPothuganti) [reply Components/Card/card.html]: Thankyou, will do the changes.

File: Components/Card/card.css (hunk 2 above)
Comment (MaruthiPothuganti) [reply Components/Card/card.css]: Will make the changes.

**Pull request merged**